shared/macros.jinja

```diff
@@ -546,3 +546,13 @@ ocil_clause: "the correct value is not returned"
         <a xmlns='http://www.w3.org/1999/xhtml' href='{{{ link }}}'>{{{ link }}}</a>
     {{%- endif %}}
 {{%- endmacro %}}
+
+
+{{% macro body_of_warning_about_dependent_rule(rule_id, why) -%}}
+        When selecting this rule in a profile, 
+        {{%- if why %}}
+            make sure that rule with ID {{{ weblink(link="#xccdf_org.ssgproject.content_rule_" + rule_id, text=rule_id) }}} is selected as well: {{{ why }}}
```

Comment:
@matejak does `{{{ weblink(link="#xccdf_org.ssgproject.content_rule_" + rule_id, text=rule_id) }}}` work for both Datastream and XCCDF?

Reply:
I suppose so, what would be the catch? It works for datastream-powered report for sure.

Reply:
I think the catch is that in a pure XCCDF Benchmark, the rules don't have the prefix `xccdf_org.ssgproject.content_rule_`, they are just `rule_id`.

Reply:
So what is better? 
- Alink that does nothing, but that is correct in the recommended use case, or 
- no link at all, only text short rule ID?

Reply:
I would probably say a short rule ID because you would have a bug in the XCCDF-only content if you use a link.

Reply:
Yeah, in the past we have gone with short rule ID, see https://github.com/ComplianceAsCode/content/pull/3113.

Reply:
OK then - fixed by rebase.